source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.h

```diff
@@ -172,8 +188,10 @@ class JsonTranscoderFilter : public Http::StreamFilter, public Logger::Loggable<
                               Http::ResponseHeaderOrTrailerMap& trailers);
   bool hasHttpBodyAsOutputType();
   void doTrailers(Http::ResponseHeaderOrTrailerMap& headers_or_trailers);
+  void initPerRouteConfig();
 
   JsonTranscoderConfig& config_;
+  const JsonTranscoderConfig* per_route_config_{nullptr};
```

Comment:
nit: `const JsonTranscoderConfig* per_route_config_{};` is equivalent and looks like is more favored in the code base

Reply:
sounds good, changed in the whole file so it's all consistent